### PR TITLE
rspec, rspec_rails

### PR DESCRIPTION
**Pull request by mtrojanowska**

### CODE REVIEW

**Comment by mtrojanowska** [comment]
:)

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  Exclude:
+    - config/**/*
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/Alias:
+  Enabled: false
+
+Style/HashLikeCase:
```
Comment (ktopolski) [inline .rubocop.yml:30]: I'd enable this one - it's good practice to replace case-when instructions with hash.fetch. Sometimes it's even bonus points on recruitment tasks, depends on the company :)

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Style/Documentation:
+  Enabled: false
+
+RSpec/MultipleExpectations:
+  Enabled: false
+
+RSpec/LetSetup:
+  Enabled: false
+
+Rails/LinkToBlank:
```
Comment (ktopolski) [inline .rubocop.yml:42]: just FYI: there used to be [this vulnerability](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/), but modern browsers now add it by default, so this cop is obsolete now. No need to change anything here, just quick lesson to learn :)

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  Enabled: false
+
+RSpec/ExampleLength:
+  Enabled: false
+
+RSpec/VerifiedDoubles:
```
Comment (ktopolski) [inline .rubocop.yml:48]: I'd enable this one, it's a good practice :) no worries to learn the diff between `double` and `instance_double` just now, but it's worth having this check enabled for the future.

File: .rubocop.yml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    - 'config/**/*.rb'
+
+Layout/FirstHashElementIndentation:
+  EnforcedStyle: consistent
+
+Lint/AmbiguousBlockAssociation:
+  Exclude:
+    - 'spec/**/*_spec.rb'
+
+Metrics/AbcSize:
```
Comment (ktopolski) [inline .rubocop.yml:70]: I'd enable this one, it's good practice. More info: https://www.rubydoc.info/gems/rubocop/0.27.0/RuboCop/Cop/Metrics/AbcSize

File: personal.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
+"ruby.rubocop.executePath": "/home/marta/.rbenv/shims/",
```
Comment (ktopolski) [inline personal.rb]: let's remove it from this PR, it's personal :)

**Review by ktopolski — CHANGES_REQUESTED** [review]
Hey 👋 

Sorry for taking so long! I've left couple of quick tweaks that could improve your learning process and developer experience in the future. Please take a look at them(and remove `personal.rb`) 😄

File: personal.rb (hunk 5 above)
Comment (ktopolski) [reply personal.rb]: @mtrojanowska this is the only change blocking this PR from being merged

Comment (mtrojanowska) [reply personal.rb]: fixed

**Pull request merged**